packages/aragon-wrapper/src/rpc/handlers/path.js

```diff
@@ -0,0 +1,3 @@
+export default function (wrapper) {
+    return wrapper.path
+}
```

Comment:
It probably makes sense to use a similar pattern to the [`cache`](https://github.com/aragon/aragon.js/blob/master/packages/aragon-wrapper/src/rpc/handlers/cache.js#L3) handler here (`get` vs. `request` path).